metadata-ingestion/setup.py

```diff
@@ -377,6 +377,7 @@ def get_long_description():
     "powerbi-report-server": powerbi_report_server,
     "vertica": sql_common | {"vertica-sqlalchemy-dialect[vertica-python]==0.0.1"},
     "unity-catalog": databricks | sqllineage_lib,
+    "airbyte": {"requests"},
```

Comment:
requests is a common library, check if it might be already there in this setup.py

Reply:
No, If you see dbt-cloud and plusar source they are also defined in same way.